packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -112,6 +112,11 @@ const propTypes = {
    * rendered to allow for row selection to occur.
    */
   hasSelectableRows: PropTypes.bool,
+
+  /**
+   * Boolean indicating that cells without any content will be displayed as a dash, meaning "No Result."
+   */
+  hasNoResultCells: PropTypes.bool,
```

Comment:
Do we need to specify the default prop here as well?

Reply:
This prop has been removed as part of addressing Charles' comment